### DISCUSSION TITLE
例子中是否可以支持大于10B的语言模型，例如Qwen

### DISCUSSION THREAD

**Discussion by xuman2019** [star-whale/starwhale#3141] [Q&A]
目前的例子都是小于10B的模型，是否支持较大的模型，例如Qwen72B

**Comment by jialeicui** [comment]
starwhale 的 demo 为了展示平台功能, 都是比较易于启动或者看到效果的  
另外您提到的 "支持" 指的是 serve 么?  这方面的话可以参考[官方的文档](https://github.com/QwenLM/Qwen?tab=readme-ov-file#deployment), 使用 vLLM 或者官方的 Web UI 集成进来
